runtime-manager/modules/ROOT/pages/rtm-traffic-inspection.adoc

```diff
@@ -0,0 +1,111 @@
+= Traffic Inspection for Standalone Mules
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: agent, runtime manager, traffic inspection, standalone
+:page-deployment-options: hybrid
+
+The Traffic Inspection feature for Mules adds support in the agent for a forward proxy that is deployed in customers' premises and acts as a man in the middle between the Mule and the control plane by intercepting and inspecting all HTTPS traffic. 
+
+== Prerequisites 
+
```

Comment:
Our style is to not have two headings in a row without content.

Reply:
I think you can remove the `=== Traffic Inspection Proxy` heading because there's only one heading in the Prereqs - no need to distinguish. 

Once you revise the bullet list to be more paragraphy, you can make it clear that the prereqs are for the Proxy.